update/sync/syncTransactions_test.go

```diff
@@ -0,0 +1,128 @@
+package sync
+
+import (
+	"encoding/json"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	dataTransaction "github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/update/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func createMockArgs() ArgsNewPendingTransactionsSyncer {
+	return ArgsNewPendingTransactionsSyncer{
+		DataPools:      mock.NewPoolsHolderMock(),
+		Storages:       &mock.ChainStorerMock{},
+		Marshalizer:    &mock.MarshalizerFake{},
+		RequestHandler: &mock.RequestHandlerStub{},
+	}
+}
+
+func TestNewPendingTransactionsSyncer(t *testing.T) {
+	t.Parallel()
+
+	args := createMockArgs()
+	args.Storages = &mock.ChainStorerMock{GetStorerCalled: func(unitType dataRetriever.UnitType) storage.Storer {
+		return &mock.StorerStub{}
+	}}
```

Comment:
put `GetStorerCalled` on a new line and reformat

Reply:
done.